docs/metrics/metrics-operators/along.md

```diff
@@ -10,7 +10,7 @@ keywords:
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-The `along metrics` operator is useful when you join queries – it allows you to control what results are joined based on the value of one or more result fields. For more information, see [Join Query Results](docs/metrics/introduction/joins.md).
+The `along` metrics operator is useful when you join queries – it allows you to control what results are joined based on the value of one or more result fields. For more information, see [Join Query Results](docs/metrics/introduction/joins.md).
```

Comment:
Formally, `along` is not an operator, it's just a statement.

Reply:
@betenkowski can you help us re-word that paragraph?

Reply:
*The `along` statement is useful...* will work.